eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -142,7 +148,10 @@ public function load(array $configs, ContainerBuilder $container)
      */
     public function getConfiguration(array $config, ContainerBuilder $container)
     {
-        return new Configuration($this->getMainConfigParser(), $this->suggestionCollector);
+        $configuration = new Configuration($this->getMainConfigParser(), $this->suggestionCollector);
+        $configuration->setSiteAccessConfigurationFilters($this->siteaccessConfigurationFilters);
```

Comment:
Well, in this context not really.
More generally, it's just that setter injection is optional, so you can't be sure that what's supposed to be injected is really here.